src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/interior_data_cache_impl.cc

```diff
@@ -88,6 +94,22 @@ void InteriorDataCacheImpl::ClearCache() {
 void InteriorDataCacheImpl::ResetRequestCountOnTimer() {
   LOG4CXX_AUTO_TRACE(logger_);
   amount_of_request_in_this_time_frame_ = 0;
+  std::this_thread::sleep_for(
```

Comment:
@ZhdanovP why do you sleep in the body of the timer? The timer will periodically call this function. Periodically means once a time frame. We do not need to sleep

Reply:
@LuxoftAKutsan removed ff6979d4bdcbd3dca9dd6a79d9a02e83c4c40788